File: gin/entity/po/Category.go
package po

type Category struct {
	ID         int    `gorm:"column:id" json:"id"`
	Name       string `gorm:"column:name" json:"name"`
	CreateTime string `gorm:"column:create_time" json:"createTime"`
	UpdateTime string `gorm:"column:update_time" json:"updateTime"`
}

func (Category) TableName() string {
	return "category"
}

File: fastapi/app/internal/models/category.py
from app.core.db import Base
from sqlalchemy import Column, DateTime, Integer, String


class Category(Base):
    """分类实体类"""

    __tablename__ = "category"
    id = Column(Integer, primary_key=True, index=True, autoincrement=True)
    name = Column(String(255), nullable=False)
    create_time = Column(DateTime, nullable=True)
    update_time = Column(DateTime, nullable=True)
